source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,94 +2,56 @@
 
-	docker run --publish 9090:9090 \
-	           --volume "$PWD/prometheus.yml:/etc/prometheus/prometheus.yml" \
-	           prom/prometheus
-	```
-	where API_ENDPOINT is:
-	- `https://metrics.cloud.service.gov.uk/metrics` for the Ireland region
-	- `https://metrics.london.cloud.service.gov.uk/metrics` for the London region
+We recommend that this Cloud Foundry account:
 
-1. Make the script executable:
+- uses the [`SpaceAuditor` role](/orgs_spaces_users.html#space-auditor) as this role has the minimum permissions needed to meet the requirements of the metrics exporter app
+- is separate to your primary Cloud Foundry account
 
-	```
-	chmod +x test-metrics.sh
-	```
+To set up the PaaS Prometheus exporter app:
 
-1. Execute the script:
+1. Clone the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter).
+1. [Push the PaaS Prometheus exporter app](/deploying_apps.html#deploying-public-apps) to Cloud Foundry without starting the app by running `cf push --no-start prometheus-exporter`.
+1. Set the following mandatory environment variables in the metrics exporter app by running `cf set-env prometheus-exporter NAME VALUE`:
 
-	```
-	./test-metrics.sh
-	```
+	|Name|Value|
+	|:---|:---|
+	|`API_ENDPOINT`|- `https://api.cloud.service.gov.uk` for Ireland<br>- `https://api.london.cloud.service.gov.uk` for London|
+	|`USERNAME`|Cloud Foundry User|
+	|`PASSWORD`|Cloud Foundry Password|
 
-	If the script executes successfully, you will see the message:
+	You should use the `cf set-env` command for these mandatory variables as they contain secret information, and this method will keep them secure.
 
-	```
-	msg="Server is ready to receive web requests."
-	```
+	You can also set environment variables by amending the manifest file. We recommend that you use this method for optional environment variables that do not contain secret information. Refer to the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter) for more information.
 
-1. Open your web browser and go to `http://localhost:9090/targets`. You should see the local Prometheus instance running in a Docker container and receiving metrics.
+1. Start the metrics exporter app by running `cf start prometheus-exporter`.
 
-If your local Prometheus instance is not receiving any metrics, check that the __PaaS State__ is __UP__.
+You can now check your monitoring system to see if you are receiving metrics.
 
-If the __PaaS State__ is __UP__ and you are still not receiving any metrics, contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+If you are not receiving any metrics, check the [logs](/monitoring_apps.html#logs) for the metrics exporter app. If you still need help, please contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
```

Comment:
This service would expose the application and metrics endpoint publically, without any authentication.  I wonder if we should mention it. 

As a solution, IMO the right solution shall be that the `paas-prometheus-exporter` does do basic HTTP auth, and that would be easy to implement. But we didn't bother ro implement that yet!

Meanwhile that is not implemented, we could suggest to use a authentication route service like the one proposed in: https://docs.cloud.service.gov.uk/deploying_services/route_services/#example-route-service-to-add-authentication